neutronutils/utils.go

```diff
@@ -0,0 +1,11 @@
+package neutronutils
```

Comment:
I don't think I like the dedicated package for such sudo call specific stuff as limited cached context creation and out of gas panic to error conversion. please consider moving it to contractmanager module:
- make `CreateCachedContext` a method of the `ContractManagerKeeper`, maybe use `Params.SudoCallGasLimit` inside instead of accepting it as a parameter, and maybe rename to smth like `SudoCachedContext`
- move the `OutOfGasRecovery` to the `contractmanager` package and also give it a more Sudo call related name

Reply:
or even better: why do we even expose all these `contractmanager` specific stuff? why don't encapsulate both the 1) limited cached context and 2) out of gas panic in the Sudo methods of the `ContractManagerKeeper`, and handle panics and call `writeFn` inside?

Reply:
good thoughts, i've opened a dedicated pr https://github.com/neutron-org/neutron/pull/313 to make the changes.